demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -49,7 +49,7 @@ interface CallingExtensionsContract {
 
 // @TODO Move it to CallingExtensions and export it once migrated to typescript
 class CallingExtensionsWrapper implements CallingExtensionsContract {
-  private _cti: CallingExtensions;
+  _cti: CallingExtensions;
```

Comment:
I renamed the getter function to `contract` for now since we have CallingExtensionsWrapper and CallingExtensionsContract but feel free to suggest a better name. Cc @HubSpot/calling-extensions-fe